include/hip/hip_runtime_api.h

```diff
@@ -1089,7 +1089,10 @@ typedef enum hipMemAccessFlags {
     hipMemAccessFlagsProtReadWrite = 3   ///< Set the address range read-write accessible
 } hipMemAccessFlags;
 /**
- * Memory access descriptor
+ * Memory access descriptor structure is used to specify memory access
+ * permissions for a virtual memory region in Virtual Memory Management API.
+ * This structure sets access and different read, and write permissions for
+ * specific memory regions.
```

Comment:
```suggestion
 * This structure changes read, and write permissions for
 * specific memory regions.
```
[NIT] Odd sentence, needs re-writing